code/modules/mob/living/basic/space_fauna/paper_wizard/paper_wizard.dm

```diff
@@ -0,0 +1,196 @@
+/mob/living/basic/paper_wizard
+	name = "Mjor the Creative"
+	desc = "A wizard with a taste for the arts."
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	faction = list(FACTION_HOSTILE, FACTION_STICKMAN)
+	icon = 'icons/mob/simple/simple_human.dmi'
+	icon_state = "paperwizard"
+	gender = MALE
+
+	response_help_continuous = "brushes"
+	response_help_simple = "brush"
+	response_disarm_continuous = "pushes"
+	response_disarm_simple = "push"
+	basic_mob_flags = DEL_ON_DEATH
+
+	maxHealth = 1000
+	health = 1000
+	melee_damage_lower = 10
+	melee_damage_upper = 20
+	obj_damage = 50
+	attack_sound = 'sound/hallucinations/growl1.ogg'
+	ai_controller = /datum/ai_controller/basic_controller/paper_wizard
+	///spell to summon minions
+	var/datum/action/cooldown/spell/conjure/wizard_summon_minions/summon
+	///spell to summon clones
+	var/datum/action/cooldown/spell/pointed/wizard_mimic/mimic
+	///the loot we will drop
+	var/static/list/dropped_loot = list(/obj/effect/temp_visual/paperwiz_dying)
+
+
+/mob/living/basic/paper_wizard/Initialize(mapload)
+	. = ..()
+	apply_dynamic_human_appearance(src, mob_spawn_path = /obj/effect/mob_spawn/corpse/human/wizard/paper)
+	grant_abilities()
+	grant_loot()
+	RegisterSignal(src, COMSIG_MOVABLE_PRE_MOVE, PROC_REF(animate_step))
```

Comment:
We have an element for this, `/datum/element/effect_trail`
Unfortunately it is written incorrectly such that you are required to subtype it, you might need to also refactor it into a bespoke element 😔

Reply:
i must refacter in this pr? or i can later?

Reply:
nvm i learnedt how